src/components/FwbBlockquote/FwbBlockquote.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <blockquote :class="blockquoteClasses" :cite="cite">
+        <slot></slot>
+    </blockquote>
+</template>
+
+<script lang="ts" setup>
+import { defineProps } from 'vue'
+import { useBlockquoteClasses } from './composables/useBlockquoteClasses'
+import type { BlockquoteSize, BlockquoteAlign, BlockquoteType } from './types'
+
+interface BlockquoteProps {
+    type?: BlockquoteType;
+    size?: BlockquoteSize;
+    align?: BlockquoteAlign;
+    cite?: string;
+}
+
+const props = withDefaults(defineProps<BlockquoteProps>(), {
+    type: 'default',
+    size: 'md',
+    align: 'left',
```

Comment:
size and alignment should be changed just by overriding tailwind classes... and not by dedicated props.

(again, take a look at `src/components/Typography/FwbP.vue` to see how it's done there)

Reply:
I'm asking to be sure if what you mean here and in the messages below is this: the user should change states such as size and align by writing them in the class, not with props, and these props should be removed.
Can you please correct me if I misunderstood?